design-docs/improved-console.md

```diff
@@ -0,0 +1,283 @@
+# Improved Console Output
+
+## Primary Goals
+* Improve perceived performance by making user aware of background work
+* Modern feel
+* User extensibility
+
+### User Goals for Console
+In priority order, the user wants to answer the following questions:
+
+* When will my build finish?
+    * Soon (should I keep watching it) or later ("I'll check back")
+* Is there anything I should know about my build right now? (e.g. tests have failed, warnings)
+* What is Gradle doing right now?
+* Is my build successful?
+* What command did I execute?
+
+## Story: Display parallel work in-progress independently by operation
```

Comment:
From what I understand this story lays the ground work for customizing individual portions of the new console display layout. I'd try to reflect that in the title. Follow up stories describe the exact look & feel + behavior. Is this story meant to be a general description without the actual implementation? If we do implement something in this story how will the individual portions of `RichConsole` look like.